### PR TITLE
feat(vtkrenderwindowinteractor): added mouse enter and mouse leave ev…

### PR DESCRIPTION
**Pull request by gandis0713**
How about adding two events to RenderWindowInteractor to use mouse enter and mouse leave events in InteractorStyle or Widget?

### CODE REVIEW

File: Sources/Rendering/Core/RenderWindowInteractor/index.js [hunk 1]
```diff
@@ -208,8 +210,9 @@ function vtkRenderWindowInteractor(publicAPI, model) {
         model.container[invMethod]('mousemove', publicAPI.handleMouseMove);
       }
 
+      rootElm[method]('mouseenter', publicAPI.handleMouseEnter);
+      rootElm[method]('mouseleave', publicAPI.handleMouseLeave);
```
Comment (jourdain) [inline Sources/Rendering/Core/RenderWindowInteractor/index.js:214]: They should not be added to the document.

**Review by jourdain — CHANGES_REQUESTED** [review]
Since you add/remove the handler in the bind/unbind you should not do it at the document level inside the `interactionRegistration` method.

File: Sources/Rendering/Core/RenderWindowInteractor/index.js (hunk 1 above)
Comment (gandis0713) [reply Sources/Rendering/Core/RenderWindowInteractor/index.js]: Removed them in `interactionRegistration`.

**Comment by jourdain** [comment]
Thanks for your contribution...

**Pull request merged**